modules/ossm-security-mtls.adoc

```diff
@@ -23,14 +23,28 @@ kind: ServiceMeshControlPlane
 spec:
   version: v2.0
   security:
-    controlPlane:
+    dataPlane:
       mtls: true
 ----
 
+You can also set this in the {product-title} web console
+
+. Log in to the web console.
+
+. Click the *Project* menu and choose the `istio-system` project from the list.
+
+. Click *Operators* -> *Installed Operators*.
+
+. Click on *Service Mesh Control Plane* under *Provided APIs*.
+
+. Click the name of your `ServiceMeshControlPlane` resource, for example `production`.
+
+. One the Details page, click the toggle in the Security section for *Data Plane Security*.
+
 [id="ossm-security-mtls-sidecars-incoming-services_{context}"]
 === Configuring sidecars for incoming connections for specific services
 
-You can also configure mTLS for individual services or namespaces by creating a policy.
+You can also configure mTLS for individual services or namespaces by creating a `PeerAuthentication` resource in your dataplane namespace. 
```

Comment:
Don't need, "in your dataplane namespace."

Reply:
Might also want to add that if setting PeerAuthentication to STRICT and not using autoMTLS, a DestinationRule is also required for the service(s).